WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -466,7 +466,7 @@ private async Task FailFastTransactionSigningPhaseAsync(Round round, Cancellatio
 
 		round.LogInfo($"Removed {removedAlices} alices, because they weren't ready. Remaining: {round.InputCount}");
 
-		if (round.InputCount >= round.Parameters.MinInputCountByRound)
+		if (round.InputCount >= Config.MinInputCountByBlameRound)
```

Comment:
Never use `Config` anywhere else except in the round creation. All comparisons must be done against the round parameters and not against the config. Unfortunately this means that you would need to add the multiplier to the round parameter, what it not possible without ruining everything because it is a type shared with the client. 

So, the solution could be adding a method/propertyGetter to `Round` like `EnoughInputsForABlameRound()` which compares the `InputCount` against a local round field `minInputCountForBlameRound` and returns true or false.

Then, you can create a blame round based on the previous failed round.